tideways/provision.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+# Tideways with XHgui
+DIR=`dirname $0`
+
+install_tideways() {
+    # Tideways is only for php =>7.0
+    if [[ ! -d /var/local/tideways-php/.git ]]; then
+        echo "Cloning Tideways extension"
+        git clone "https://github.com/tideways/php-xhprof-extension" /var/local/tideways-php
+    else
+        echo "Updating Tideways extension"
+        ( cd /var/local/tideways-php/ && git pull -q && git checkout -q )
+    fi
+}
+
+install_tideways_php() {
+    version=$1
+    if [[ `command -v php$version` ]]; then
+        echo "Copying tideways files for php $version"
+        cp -f "${DIR}/tideways.ini" "/etc/php/$version/mods-available/tideways_xhprof.ini"
+        cp -f "${DIR}/mongodb.ini" "/etc/php/$version/mods-available/mongodb.ini"
```

Comment:
I am getting some errors about this file not avalaible when mongodb is installed. 
I have to move it in the mongo install function

Reply:
+1